### DISCUSSION TITLE
Dark Theme is not fully dark

### DISCUSSION THREAD

**Discussion by nsgnkhibdk2cls0f** [stefankueng/BowPad#299] [General]
The Ribbon is grayish with white texts so the contrast is low and difficult to read. Especially the drop down lists like Language or Lexer, it's still too light. The right click menu is also grayish. The Ribbon is more lighter than the rest of the UI caused inconsistency and will hurt the eyes if you reading code too long and already used to the darker theme here and immediately switch to use the Ribbon.

**Comment by stefankueng** [comment]
are you still using Win7?

**Reply by nsgnkhibdk2cls0f** [reply]
Windows 8.1

**Comment by stefankueng** [comment]
Well, up until Win10 the ribbon does not really have a dark mode, so for those older OS versions the dark mode is emulated by setting the ribbon colors using another way. Problem is that the colors can't be fully set, they're always blended by the OS.

Also, since Win8.1 has reached end-of-life six days ago, I strongly recommend you update your OS.

**Reply by nsgnkhibdk2cls0f** [reply]
Could you try to make it a little darker to be consistent with the rest of the UI on Windows 8.1 and Windows 7? Office 2010 and Office 2016 have a nice dark theme on Windows 8.1 and Windows 7. I recall I always use the Silver theme of Office 2010 on Windows 7. Something like the Silver theme is already good enough for me. If you could clone the Silver theme of Office 2010 it's too good.

**Comment by stefankueng** [comment]
as I said, the ribbon for older OSes does not provide that flexibility for setting its color. I made the colors as good as possible. It can't get any better. Sorry.

**Reply by nsgnkhibdk2cls0f** [reply]
I understand. I will use light theme as I can't upgrade Windows.
